requirements.txt

```diff
@@ -6,3 +6,5 @@ watchdog==0.8.3
 # Use sckoo@stanford.edu's patched argcomplete, until pull request has been accepted
 # https://github.com/kislyuk/argcomplete/pull/118
 git+https://github.com/kashizui/argcomplete.git@allow-overridden-subparsersaction
+# Use agd@stanford.edu's fork of hash_ring with extra features
+git+https://github.com/andreweduffy/hash_ring.git@master
```

Comment:
This is kind of unfortunate...what features do we need?
Can we just put the code directly into CodaLab?
I don't think it's good having tons of dependencies if they're going to be custom anyway.